sdk/src/types/api/core.rs

```diff
@@ -24,6 +24,24 @@ use crate::{
     utils::serde::{option_string, string},
 };
 
+/// Response of GET /api/routes.
+/// The available API route groups of the node.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RoutesResponse {
+    pub routes: Vec<String>,
+}
+
+/// Contains the info and the url from the node (useful when multiple nodes are used)
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NodeInfoResponse {
```

Comment:
Idk about that tbh, I would keep this file for the actual TIP responses, this type is purely from us

Reply:
@Thoralf-M can you give you input as well, pls?

Reply:
I have no problem with having the struct in there, the same way as I'm fine with having the method that returns this wrapper together with the core routes :shrug:

Reply:
@thibault-martinez any update on this?